tsconfig.json

```diff
@@ -25,6 +25,7 @@
     "inlineSources": true,
     // Prevents web types from being suggested by vscode.
     "types": ["node"],
+    "forceConsistentCasingInFileNames": true
```

Comment:
I don't expect us to ever make a file with capital letters in it but this is some free linting to prevent potential issues on windows